redwood-lazylayout-schema/src/main/kotlin/app/cash/redwood/lazylayout/widgets.kt

```diff
@@ -22,6 +22,8 @@ import app.cash.redwood.schema.Widget
 @Widget(1)
 public data class LazyList(
   @Property(1) val isVertical: Boolean,
-  @Property(2) val onPositionDisplayed: (position: Int) -> Unit,
+  @Property(2) val onViewportChanged: (range: IntArray) -> Unit,
```

Comment:
Multiple event arguments are supported now if you want to explicitly declare the two here instead of this more general structure.